### DISCUSSION TITLE
请问AntDesign目前支持CSS变量方式定制主题吗？

### DISCUSSION THREAD

**Discussion by likuner** [ant-design/ant-design#37128] [General]
### What problem does this feature solve?

想要使用CSS变量方式定制主题

### What does the proposed API look like?

想问有没有相关API

**Comment by zombieJ** [comment]
只支持主题色，下个大版本将通过 cssinjs 支持全量动态主题：
https://ant.design/docs/react/customize-theme-variable-cn
